flogin/jsonrpc/client.py

```
@@ -40,10 +44,8 @@ def request_id(self) -> int:
     def request_id(self, value: int) -> None:
         self._current_request_id = value
 
-    async def request(
-        self, method: str, params: list[object] = []
-    ) -> Any | ErrorResponse:
-        fut: asyncio.Future[Any | ErrorResponse] = asyncio.Future()
+    async def request(self, method: str, params: list[object] = []) -> Any:
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)